app/views/survey/show.erb

```diff
@@ -1 +1,17 @@
-<h1>show</h1>
+<h2> Survey: <i><%= @survey.title %></h2>
+
+<form action="/survey/<%= @survey.id %>" method="post">
+
+  <ol>
+    <% index = 0 %>
+    <% @survey.questions.each do |question| %>
+      <li> <%= question.name%> </li>
+      <ul><% question.choices.each do |choice| %></ul>
+          <input type="radio" name="choices[:id]" value="<%= choice.id %>"> <%=choice.body %><br>
```

Comment:
having a <br> tag in the html is usually a code smell I believe. I am not sure how to fix this or what the actual way to do this is but this but I think this line could be cleaned . For now as a long as it works and loooooks pruuuuuty I am fine with leaving it in the MVP
:sunglasses: :palm_tree: :monkey: